integrations/extensions/docs/elasticsearch-install-and-setup/README.md

```diff
@@ -0,0 +1,69 @@
+# Elasticsearch Installation and Setup Documentation
+
+This directory contains documentation for installing and setting up Elasticsearch along with related tools and integrations.
+
+## Elasticsearch Setup
+- [Guide to installing Docker or Docker alternatives](how_to_install_docker.md): A guide explaining Docker and Docker Compose installation options, essential for running Elasticsearch-related applications.
+- [Set up Elasticsearch from IBM Cloud and integrate it with Watson Assistant](ICD_Elasticsearch_install_and_setup.md): Instructions for provisioning Elasticsearch instance on IBM Cloud and setting up related services like Kibana.
+- [Set up watsonx Discovery (Elasticsearch) and integrate it with Watson Assistant in CloudPak](watsonx_discovery_install_and_setup.md): Documentation for setting up Watsonx Discovery (Elasticsearch) and integrating it with Watson Assistant in CloudPak.
+
+## Elasticsearch integration with Watson Assistant
+### 1. Set up a custom Elasticsearch extension
+You can follow the guide [here](../../starter-kits/elasticsearch/README.md) to build a custom Elasticsearch extension allowing your assistant to search for information in an Elasticsearch index and show what it finds in the chat.
+### 2. Set up the built-in Elasticsearch extension
+You can set up search over an Elasticsearch index using the built-in Elastic search extension as well. Please follow the instructions [here](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-elasticsearch-add) to set up the extension for your assistant. You can then enable search for your extension using ELSER or dense vector search by following the relevant instruction below 
+
+#### a. ELSER
+For semantic search using ELSER v1, please include the following query body in the Search extension settings:
+
+```json
+{
+  "query":{
+    "text_expansion":{
+      "ml.tokens":{
+        "model_id":".elser_model_1",
+        "model_text":"$QUERY"
+      }
+    }
+  }
+}
+```
+
+![](./assets/query_body_for_elasticsearch.png)
```

Comment:
You may need to specify the height and width of this image. I would set it to `original_height / 2.5`

Reply:
Thanks, Zach! Added image height and width.